focus-block/src/Components/App/App.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import NewBlockForm from '../NewBlockForm/NewBlockForm'
+import ActiveBlock from '../ActiveBlock/ActiveBlock';
+
+class App extends Component {
+  constructor(props) {
+		super(props);
+		this.state = {
+      title: 'New Task',
+      isAltLogoColor: false,
+      wrapperClass: '',
+      currentBlock: null
+    };
+  }
+
+  formSubmitHandler = (block) => {
+    this.setState({currentBlock: block});
+  }
+
+  appearanceChangeHandler = (title, isAltLogoColor, isRedOutline, isPopup, isFinished) => {
+    let wrapperClass = '';
+    if (isRedOutline) {
+      wrapperClass = 'app-wrapper--times-up';
+    } else if (isPopup) {
+      wrapperClass = 'app-wrapper--popup'
+    } else if (isFinished) {
+      wrapperClass = 'app-wrapper--finished'
+    }
+    this.setState({
+      title: title,
+      isAltLogoColor: isAltLogoColor,
+      wrapperClass: wrapperClass
+    });
+  }
+
+  blockEndHandler = () => {
+    this.setState({
+      title: 'New Task',
+      isAltLogoColor: false,
+      wrapperClass: '',
+      currentBlock: null
+    });
+  }
+  
+  render() {
+    return(
+      <div className={'app-wrapper ' + this.state.wrapperClass}>
+        <div className="app-header">
+          <div className="app-header__logo">
+            <img 
+              src={this.state.isAltLogoColor ? 
+              require("../../Assets/focusblock-logo-blue.svg")
```

Comment:
Lets require these svgs at the top of the file and then just pass in the prop names.

For example
`blueLogo = require("../../Assets/focusblock-logo-blue.svg")`
`redLogo = require("../../Assets/focusblock-logo-red.svg")`

Reply:
Duplicate of global "requirements should be at the top" comment.